tests/unit/cache_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2018 - Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit
+
+import (
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/okta/okta-sdk-golang/okta/cache"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_cache_key_can_be_created_from_request_object(t *testing.T) {
```

Comment:
Can we also test the request executor?  It would be good to assert that for a URL, the request executor returns that item from the cache if it exists in the cache.

Reply:
Will be implemented in the unit testing pass